equinix/resource_metal_spot_market_request_acc_test.go

```diff
@@ -93,7 +93,7 @@ data "equinix_metal_spot_market_request" "dreq" {
 
 resource "equinix_metal_spot_market_request" "request" {
   project_id       = equinix_metal_project.test.id
-  max_bid_price    = data.equinix_metal_spot_market_price.test.price * 1.2
+  max_bid_price    = data.equinix_metal_spot_market_price.test.price
```

Comment:
Here's an example of failure due to a bid price that is higher than the allowed bid price: https://github.com/equinix/terraform-provider-equinix/actions/runs/5050831197/jobs/9062032812#step:5:624

It's possible we should be extra safe and multiply the spot market price by `0.95` instead?

Reply:
...aaand it looks like we _do_ need to further reduce the max bid price, because this test still failed: https://github.com/equinix/terraform-provider-equinix/actions/runs/5072458466/jobs/9110213013#step:5:673

The spot market data source tests have a hard-coded max bid of `0.01`, and I don't think I've seen those fail yet; is there some reason we're using a dynamic max bid for these tests?